### ISSUE TITLE
Tests do not pass

### ISSUE THREAD

**Issue by Lewiscowles1986** [hyperwallet/python-sdk#18]
👋 Greetings from KaloHQ

`make test` command fails

<details>
<summary>What went wrong (log)</summary>

```
======================================================================
ERROR: test_should_fail_decryption_when_wrong_private_key_is_used (hyperwallet.tests.test_encryption.EncryptionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/lewiscowles/hyperwallet/python-sdk/hyperwallet/utils/encryption.py", line 99, in decrypt
    jweToken.deserialize(body, key=privateKeyToDecrypt)
  File "/Users/lewiscowles/hyperwallet/python-sdk/.eggs/jwcrypto-0.6.0-py3.7.egg/jwcrypto/jwe.py", line 485, in deserialize
    self.decrypt(key)
  File "/Users/lewiscowles/hyperwallet/python-sdk/.eggs/jwcrypto-0.6.0-py3.7.egg/jwcrypto/jwe.py", line 412, in decrypt
    'key' + repr(self.decryptlog))
jwcrypto.jwe.InvalidJWEData: No recipient matched the provided key["Failed: [ValueError('Decryption failed.')]"]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/lewiscowles/hyperwallet/python-sdk/hyperwallet/tests/test_encryption.py", line 42, in test_should_fail_decryption_when_wrong_private_key_is_used
    encryption2.decrypt(encryptedMessage)
  File "/Users/lewiscowles/hyperwallet/python-sdk/hyperwallet/utils/encryption.py", line 101, in decrypt
    raise HyperwalletException(e.message)
AttributeError: 'InvalidJWEData' object has no attribute 'message'

======================================================================
ERROR: test_should_fail_signature_verification_when_wrong_public_key_is_used (hyperwallet.tests.test_encryption.EncryptionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/lewiscowles/hyperwallet/python-sdk/.eggs/python_jose-3.0.1-py3.7.egg/jose/jws.py", line 265, in _verify_signature
    raise JWSSignatureError()
jose.exceptions.JWSSignatureError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/lewiscowles/hyperwallet/python-sdk/hyperwallet/utils/encryption.py", line 108, in decrypt
    return jws.verify(payload, json.dumps(jwkCheckSignKey), algorithms=self.signAlgorithm)
  File "/Users/lewiscowles/hyperwallet/python-sdk/.eggs/python_jose-3.0.1-py3.7.egg/jose/jws.py", line 77, in verify
    _verify_signature(signing_input, header, signature, key, algorithms)
  File "/Users/lewiscowles/hyperwallet/python-sdk/.eggs/python_jose-3.0.1-py3.7.egg/jose/jws.py", line 267, in _verify_signature
    raise JWSError('Signature verification failed.')
jose.exceptions.JWSError: Signature verification failed.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/lewiscowles/hyperwallet/python-sdk/hyperwallet/tests/test_encryption.py", line 59, in test_should_fail_signature_verification_when_wrong_public_key_is_used
    encryption2.decrypt(encryptedMessage)
  File "/Users/lewiscowles/hyperwallet/python-sdk/hyperwallet/utils/encryption.py", line 110, in decrypt
    raise HyperwalletException(e.message)
AttributeError: 'JWSError' object has no attribute 'message'

======================================================================
ERROR: test_should_throw_exception_when_jwk_set_file_retrieved_from_url_is_invalid (hyperwallet.tests.test_encryption.EncryptionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/lewiscowles/hyperwallet/python-sdk/.venv/lib/python3.7/site-packages/mock/mock.py", line 1330, in patched
    return func(*args, **keywargs)
  File "/Users/lewiscowles/hyperwallet/python-sdk/hyperwallet/tests/test_encryption.py", line 215, in test_should_throw_exception_when_jwk_set_file_retrieved_from_url_is_invalid
    self.assertEqual(exc.exception.message, 'expected string or buffer')
AttributeError: 'TypeError' object has no attribute 'message'

======================================================================
FAIL: test_should_successfully_encrypt_and_decrypt_text_message (hyperwallet.tests.test_encryption.EncryptionTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/lewiscowles/hyperwallet/python-sdk/hyperwallet/tests/test_encryption.py", line 27, in test_should_successfully_encrypt_and_decrypt_text_message
    self.assertEqual(decryptedMessage, testMessage)
AssertionError: b'Message for test' != 'Message for test'

----------------------------------------------------------------------
```

</details>

**Comment by wmews-hw** [comment]
Hey @Lewiscowles1986,
We've merged #19 in order to fix the issues you pointed out.

**Comment by Lewiscowles1986** [comment]
Thanks so much. May have some PR's flowing in. It looks like someone forked this at 0.2.0, didn't set upstreams or by the looks of things open any PR's.

Of course this has high coverage so this may be better than the PR's

Thank you!
